src/interface/storage.thrift

```diff
@@ -203,6 +205,44 @@ struct CatchUpDataReq {
     3: common.HostAddr     target,
 }
 
+struct UpdateRespData {
+    1: bool upsert,
+    2: common.VertexID vertex_id,       // Only valid when update vertex
+    3: EdgeKey edge_key,                // Only valid when update edge
+    4: binary data,
+}
+
+struct UpdateResponse {
+    1: required ResponseCommon result,
+    2: optional common.Schema schema,   // return column related props
+    3: optional list<UpdateRespData> return_data,
+}
+
+struct UpdateItem {
+    1: required binary name,    // Tag or EdgeType schema name
+    2: required binary field,   // property name of scheam
+    3: required binary value,   // expression which is encoded
+}
+
+struct UpdateVertexRequest {
+    1: common.GraphSpaceID space_id,
+    2: map<common.PartitionID, list<common.VertexID>>(cpp.template = "std::unordered_map") parts,
```

Comment:
I am not sure how to support update batch of vertex in one update sentence

Reply:
It already changed to a single update!